oasis/main.setup.ml.in

```diff
@@ -0,0 +1,20 @@
+
+let build_id () : unit =
+  BaseEnv.var_define
```

Comment:
Don't use the `BaseEnv.var_define` to define variables but our own specialized `add_variable` function.